modules/openxAnalyticsAdapter.js

```diff
@@ -237,6 +244,106 @@ function removeads(info) {
   }
 }
 
+function getAuctionIdByAdId(adId) {
+
+  let auctionId;
+  utils._map(eventStack, value => value).forEach( function(auctionInfo) {
```

Comment:
Just a nitpick: you dont have to do a _map, when there is a _each function in the utils that does the same thing as a map/forEach.